src/cfml/system/services/ServerService.cfc

```diff
@@ -681,9 +767,21 @@ component accessors="true" singleton {
 	*/
 	function saveServerJSON( required string directory, required struct data ) {
 		var filePath = arguments.directory & '/server.json';
+		if(fileExists(filePath)) {
+		  var existingProps = readServerJSON(directory);
+		  if(!isNull(data.name) && (isNull(existingProps.name) || existingProps.name != data.name)) {
+		    filePath = arguments.directory & '/server-#data.name#.json';
```

Comment:
This is something Denny did, where basically, if you just start a server normal, you get a `server.json` file, but if you start the server with the name `foo`, you get a `server-foo.json` file.  I think the thought was that you would pick a different name for each CF engine and you would get separate files by default to store the details of that server.  
I'm still mulling over how to make this work as it also needs some internal changes to how the server settings are stored to keep things like the stop port separate.  I'll finish it as part of https://ortussolutions.atlassian.net/browse/COMMANDBOX-345